api-docs/api-reference/methods/users/post-forgot-pwd-v2.0.rst

```diff
@@ -0,0 +1,92 @@
+.. _post-forgot-pwd-v2.0:
+
+Forgot Password
+~~~~~~~~~~~~~~~
+
+.. code::
+
+    POST /v2.0/users/RAX-AUTH/forgot-pwd
+
+Use the Forgot Password operation to receive an email with a password reset token.
+
+Identity users can ONLY use the token generated from the Forgot Password operation
+to reset their password via the :ref:`Reset Password <reset-password-v2.0>` operation.
+
+The generated token will be sent to the email associated with the user account.
+
+.. note::
+
+   - Restrict forgot password service to only those enabled users who have either the
+     identity:user-admin or identity:default roles. This would implicitly include
+     identity:user-manager.
+
+   - Reset Password token has a short lifetime.
+
```

Comment:
What does "a short lifetime" mean?    

I added a suggested revision in a comment above.